exercise2/Monkeys.java

```diff
@@ -1,28 +1,53 @@
-package exercise2;
 
 class Monkeys {
 
     boolean monkeyBusiness(Monkey[] monkeys) {
+        int isSmiling = 0;
+        int isFrowning = 0;
+
+        for(Monkey newMonkey : monkeys )
+            if (newMonkey.smile){
+                isSmiling++;
+            }
+            else isFrowning++;
+
+            return isSmiling >= isFrowning; //if majority are not smiling, returning false. if it's a tie, return true
     //BONUS EXERCISE
-    //We have a set of monkeys.  Each Monkey has a boolean smile property, which indicates if that particular monkey is smiling.  Determine whether or not the majority of monkeys are smiling.  If the majority are smiling, return true.  if the majority are not smiling, return false.  If it is a tie, return true.  
-        return true;
+    //We have a set of monkeys.  Each Monkey has a boolean smile property, which indicates if that particular monkey is smiling
+        // Determine whether or not the majority of monkeys are smiling.  If the majority are smiling, return true.
+        // if the majority are not smiling, return false.  If it is a tie, return true.
     }
 
     class Monkey {
         String name;
         boolean smile;
     }
 
+    public Monkey getMonkey(String name, boolean smile){
```

Comment:
a parameterized constructor would be better, unless you go with a Factory pattern